src/library/dao/cue.cpp

```diff
@@ -19,20 +21,22 @@ Cue::Cue(TrackId trackId)
           m_iPosition(-1),
           m_iLength(0),
           m_iHotCue(-1),
-          m_label("") {
+          m_label(""),
```

Comment:
Not, exactly, the default constructor creates a null string.
http://doc.qt.io/qt-4.8/qstring.html#distinction-between-null-and-empty-strings

I realized there was a problem in waveformrendermark.cpp:99
`if (newLabel != mark.m_text || newColor != mark.m_color) {`
where I was implicitly checking that mark.m_text was initialized to a value.

I've made this explicit, also for m_color.